node/actors/internal_nats_server.go

```diff
@@ -62,9 +69,9 @@ func (ns *internalNatsServer) HandleMessage(from gen.PID, message any) error {
 			return err
 		}
 		ns.tokens[InternalNatsServerReadyName] = token
-		ns.Log().Info("registered publishable event %s, waiting for consumers...", InternalNatsServerReadyName)
+		ns.Log().Info("registered publishable event, waiting for consumers...", slog.Any("atom", InternalNatsServerReadyName))
 	case eventStart:
-		ns.Log().Info("publisher got first consumer for %s. start producing events...", InternalNatsServerReadyName)
+		ns.Log().Info("publisher got first consumer. start producing events...", slog.Any("atom", InternalNatsServerReadyName))
```

Comment:
do not use `atom` here, that's meaningless to someone viewing the logs. it's an `event_name`

Reply:
Copypasta artifact.